src/components/common/form/error-summary/ErrorSummary.jsx

```diff
@@ -2,6 +2,18 @@ import PropTypes from 'prop-types';
 import { HashLink } from 'react-router-hash-link';
 
 const ErrorSummary = ({ errors }) => {
+  const callbackRef = (node) => {
+    // if node exists (there is a summary error), scroll to the first summary error and focus it
+    if (node) {
```

Comment:
Same with `node` - could that be renamed to something like `summaryError`?

Reply:
I've renamed this variable to summaryErrorsList now